build/scripts/build-inspector.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+set -e
+
+# TODO: Add as CMake Target
+
+WORKSPACE=`pwd`
+CONFIGURATION="Release"
+
+mkdir -p "$WORKSPACE/cmake-build"
+
+WEBKIT_SOURCE_PATH="$WORKSPACE/src/webkit"
+WEBKIT_BUILD_OUTPUT_PATH="$WORKSPACE/cmake-build/WebKit-Xcode"
+
+INSPECTOR_SOURCE_PATH="$WORKSPACE/src/debugging/Inspector/Inspector"
+INSPECTOR_BUILD_OUTPUT_PATH="$WORKSPACE/cmake-build/Inspector"
+
+echo "Building WebKit..."
+xcodebuild \
```

Comment:
@fealebenpae I'm experiencing some failing builds using the `build-webkit` tool. Do you have any idea of: `fatal error: 'wtf/Platform.h' file not found'`?